pkgs/by-name/ce/celeste64/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  SDL2,
+  libGL,
+  mesa,
+  fmodex,
+  systemd,
+  libpulseaudio,
+  libselinux,
+  wayland,
+  libdecor,
+  libxkbcommon,
+  libX11,
+  libXfixes,
+  libXext,
+  libXi,
+  libXrandr,
+  libXcursor,
+  withSELinux ? false,
+}:
+buildDotnetModule rec {
+  pname = "celeste64";
+  version = "0-unstable-2024-01-31";
+
+  src = fetchFromGitHub {
+    repo = "Celeste64";
+    owner = "ExOK";
+    sha256 = "sha256-S9KOtCrJSsIQxvZWjiHUYEiM9/TbO0q6vSDPwJfyW3I=";
+    rev = "62417823a30582e8561075fe0f7d4b398aedc353";
+  };
+  projectFile = "Celeste64.csproj";
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  nugetDeps = ./deps.nix;
+  strictDeps = true;
+  executables = [ "Celeste64" ];
+  nativeBuildInputs = [ copyDesktopItems ];
+  runtimeDeps =
+    [
+      mesa
+      libdecor
+      libGL
+      SDL2
+      fmodex
+      libxkbcommon
+      systemd
+      libpulseaudio
+      wayland
+      libX11
+      libXfixes
+      libXext
+      libXcursor
+      libXi
+      libXrandr
+    ]
+    ++ lib.optionals withSELinux [ libselinux ];
+
+  postFixup = ''
```

Comment:
was there a reason to not use postInstall here?

Reply:
No reason! just didn't think to use it!